iis-workflows/iis-workflows-ingest-pmc/src/test/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/PmcXmlHandlerTest.java

```diff
@@ -255,42 +348,42 @@ public void testMixedTitleParsing() throws Exception {
 			reader.setFeature("http://xml.org/sax/features/validation", false);
 			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
 			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
-			
+
 			ExtractedDocumentMetadata.Builder metaBuilder = ExtractedDocumentMetadata.newBuilder();
 			metaBuilder.setId("some-id");
 			metaBuilder.setText("");
 			PmcXmlHandler pmcXmlHandler = new PmcXmlHandler(metaBuilder);
-			saxParser.parse(inputStream = PmcXmlHandler.class.getResourceAsStream(filePath), 
-					pmcXmlHandler);
+			saxParser.parse(inputStream = PmcXmlHandler.class.getResourceAsStream(filePath), pmcXmlHandler);
 			ExtractedDocumentMetadata meta = metaBuilder.build();
 			assertNotNull(meta.getReferences());
 			ReferenceMetadata refMeta = meta.getReferences().get(12);
-			assertEquals("13 Shearer KD, Silverstein J, Plisetskaya EM (1997) Role of adiposity in food intake control of juvenile chinook salmon (Oncorhynchus tshawytscha). Comp Biochem Physiol A 118: 1209–1215",
+			assertEquals(
+					"13 Shearer KD, Silverstein J, Plisetskaya EM (1997) Role of adiposity in food intake control of juvenile chinook salmon (Oncorhynchus tshawytscha). Comp Biochem Physiol A 118: 1209–1215",
 					refMeta.getText());
 			assertEquals(new Integer(13), refMeta.getPosition());
-			assertEquals(Arrays.asList(new String[] {"Shearer, KD", "Silverstein, J", "Plisetskaya, EM"}), 
+			assertEquals(Arrays.asList(new String[] { "Shearer, KD", "Silverstein, J", "Plisetskaya, EM" }),
```

Comment:
Try `Lists.newArrayList` or ImmutableList.of` its little shorter and little easier to write